README.md

```diff
@@ -69,9 +76,9 @@ Before checking in, use the `pre-commit.sh` script to check code formatting, run
 unit tests, run integration tests etc.
 
 The integration tests need a file in your home directory called `.b2_auth`
-that contains two lines with nothing on them but your account ID and application key:
+that contains two lines with nothing on them but your application Key ID and application key:
```

Comment:
please decide on the spelling. As of now in this PR we can see it as
* application key id
* application Key ID
* application key Id

I personally prefer the first one, but the most important bit is to be consistent (so that we can one day grep the repo to find it etc)